appium/tests/helpers/ElementHelper.ts

```diff
@@ -41,7 +41,9 @@ class ElementHelper {
 
   static checkStaticText = async (element: WebdriverIO.Element, label: string) => {
     await this.waitElementVisible(element);
-    await expect(element).toHaveText(label);
+    await browser.waitUntil(
+      async () => (await element.getText()) === label
+    );
```

Comment:
This will wait infinitely as written - should be updated to newly added method once https://github.com/FlowCrypt/flowcrypt-ios/pull/1482 merges